web/src/main/java/io/fabric8/launcher/web/api/OpenShiftResource.java

```diff
@@ -89,11 +91,12 @@ public JsonArray getSupportedOpenShiftClusters(@HeaderParam(HttpHeaders.AUTHORIZ
                     .forEach(arrayBuilder::add);
         } else {
             final KeycloakService keycloakService = this.keycloakServiceInstance.get();
+            final String keycloakToken = removeBearerPrefix(authorization);
```

Comment:
Why remove the bearer to add it back in KeycloakService?

Reply:
Yes you are right, the other way around makes more sense: KeycloakService.Optional<Identity> getIdentity(String provider, String authorizationHeader)